android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -415,22 +428,30 @@ public View getView(int i, View view, ViewGroup viewGroup) {
       setText(view, R.id.description, pwsResult.getDescription());
       ((ImageView) view.findViewById(R.id.icon)).setImageBitmap(
           Utils.getBitmapIcon(mPwCollection, pwsResult));
+      final String siteUrl = pwsResult.getSiteUrl();
+        if (Utils.isFavorite(pwsResult.getSiteUrl())) {
```

Comment:
Please fix the indents here.  Also, you could use Utils.isFavorite(siteUrl) instead of calling getSiteUrl again here.